### PR TITLE
add error checks of function ds3231_get_time() and ds3231_set_time()

### PR DESCRIPTION
**Pull request by Hxinrong**
### Contribution description
tests/driver_ds3231/main.c : add error checks of function ds3231_get_time() and ds3231_set_time()
### Issues/PRs references
Addresses parts of #15524

### CODE REVIEW

File: tests/driver_ds3231/main.c [hunk 1]
```diff
@@ -94,7 +94,12 @@ static int _cmd_get(int argc, char **argv)
     char dstr[ISOSTR_LEN];
 
     struct tm time;
-    ds3231_get_time(&_dev, &time);
+    int res;
+    res = ds3231_get_time(&_dev, &time);
+    if (res != 0) {
+        puts("error: unable to read time");
```
Comment (benpicco) [inline tests/driver_ds3231/main.c:100]:
```suggestion
        DEBUG("error: unable to read time\n");
```

File: tests/driver_ds3231/main.c [hunk 2]
```diff
@@ -122,7 +127,11 @@ static int _cmd_set(int argc, char **argv)
         return 1;
     }
 
-    ds3231_set_time(&_dev, &target_time);
+    res = ds3231_set_time(&_dev, &target_time);
+    if (res != 0) {
+        puts("error: unable to set time");
```
Comment (benpicco) [inline tests/driver_ds3231/main.c:132]:
```suggestion
        DEBUG("error: unable to set time\n");
```

**Comment by aabadie** [comment]
The commit comment needs some more context, something like "tests/driver_ds3231: add error checks".

File: tests/driver_ds3231/main.c (hunk 1 above)
Comment (aabadie) [reply tests/driver_ds3231/main.c:100]: It is ok to use puts in a test application code.

Comment (benpicco) [reply tests/driver_ds3231/main.c:100]: ah I didn't realize this was the test, not the driver

**Review by benpicco — APPROVED** [review]

**Pull request merged**